index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <meta content="width=device-width, initial-scale=1.0" name="viewport">
+  <title>Set up and Mobile Version Skeletion</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" />
+
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+  <header>
+    <div class="nav-bar">
+      <a href="#">
+        My Logo
+      </a>
+      <i class="fas fa-bars"></i>
+    </div>
+  </header>
+  <main>
+    <section>
+      <div>
+        <div id="background-image"></div>
+        <div class="content">
+          <h1>I’m Mariane <br />
+            Glad to see <br /> you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+       <div class="connect">
+          <a href="#">LET’S CONNECT</a>
+        </div>
+
+          <div class="social-icons">
+          <ul>
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-linkedin"></i></a></li>
+            <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
+            <li><a href="#"><i class="fab fa-github"></i></a></li>
+            <li><a href="#"><i class="fab fa-angellist"></i></a></li>
```

Comment:
- [ ] Currently your social links have the default styling that `a` tags do and appear blue, kindly change their color so they conform to the Figma template as you can see below
> Your Design
![image](https://user-images.githubusercontent.com/90556221/170393263-88588ec7-3295-46d9-9ce9-75364bf8cf60.png)

> Figma Design
![image](https://user-images.githubusercontent.com/90556221/170393323-65df8a26-f9fa-430c-94b6-e9ebe59e8899.png)